tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -104,3 +105,32 @@ Is RHODS Version Greater Or Equal Than
     ${comparison} =  GTE  ${ver}  ${target}
     [Return]  ${comparison}
 
+Move To Installed Operator instance
+    [Arguments]   ${operator_name}     ${tab_name}      ${namespace}
+    Switch To Administrator Perspective
+    Navigate to Installed Operators
+    Installed Operators Should Be Open
+    Log To Console   ${namespace}
+    Run Keyword If  "${namespace}" == "None"   Select All Projects
+    ...         ELSE   Select Project By Name   ${namespace}
+    #Select All Projects
+    Click On Searched Operator   ${operator_name}
+    Switch To New Tab       ${tab_name}
+    sleep    5
+
+Create Installed Operator instance
+    [Arguments]    ${operator_name}     ${tab_name}     ${namespace}=None
+    Move To Installed Operator instance    ${operator_name}     ${tab_name}    ${namespace}
```

Comment:
this naming might appear as ambiguous, since the instance has not been created yet. it may be difficult for someone who's not familiar with this operation. e.g., "Go To Operator Config Page"

Reply:
have changed it to ..hope it will clear the confusion

Reply:
I didn't mean the TC title but the name of "Move to installed operator instance" keyword